src/ItemsContext.js

```diff
@@ -1,25 +1,27 @@
 import React, { createContext, useState, useEffect } from 'react';
 import firebase from 'firebase';
+import useTokenHook from './useTokenHook';
 
 export const ItemsContext = createContext();
 
 const ItemsProvider = props => {
   const [items, setItems] = useState([]);
+  const { token } = useTokenHook();
   const db = firebase.firestore();
-  const userToken = localStorage.getItem('token');
 
   useEffect(() => {
-    db.collection('items')
-      .where('token', '==', userToken)
-      .onSnapshot(querySnapshot => {
-        const list = querySnapshot.docs.map(doc => {
-          const newObj = { ...doc.data(), id: doc.id };
-          return newObj;
+    if (token) {
```

Comment:
Super grateful for you fixing our projects bug! I didn't even know where to start with this issue, but y'all, @MabelM1, and @skillitzimberg were able to figure it out. 💜